app/ts/background/settings.ts

```diff
@@ -82,17 +82,16 @@ export const defaultRpcs: RpcEntries = [
 		minimized: true,
 		weth: 0x105083929bf9bb22c26cb1777ec92661170d4285n,
 	},
-	/*
```

Comment:
this code is just commented out, you might want to reference on another issue if this one needs to be redone later